x-pack/plugins/apm/ftr_e2e/cypress_start.ts

```diff
@@ -56,21 +54,33 @@ async function cypressStart(
     },
   });
 
-  console.log('Loading esArchiver...');
-  await esArchiverLoad('apm_8.0.0');
+  const esNode = Url.format({
+    protocol: config.get('servers.elasticsearch.protocol'),
+    port: config.get('servers.elasticsearch.port'),
+    hostname: config.get('servers.elasticsearch.hostname'),
+    auth: `${config.get('servers.elasticsearch.username')}:${config.get(
+      'servers.elasticsearch.password'
+    )}`,
+  });
+
+  const esRequestTimeout = config.get('timeouts.esRequestTimeout');
+
+  console.log('Loading ES archives...');
```

Comment:
Perhaps add the name of the archive in the logs:

```suggestion
  console.log(`Loading ES archive "${archiveName}"`);
```